README.md

```diff
@@ -1,70 +1,92 @@
-# Getting Started with Create React App
+![swrpgc banner](https://user-images.githubusercontent.com/17935770/131261369-477cff64-39e5-462a-81bf-3f55944e512e.png)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+This application acts as a companion for the D&D game: **Star Wars: Edge of the Empire**. The companion app allows you to keep track of and store your in-game characters. You may also connect and communicate with your friends and party up so that everyone can witness the dice rolls that will unfold the story in their legendary campaign.
```

Comment:
Little nitpick here, I would swap 'D&D game' for 'tabletop rpg'. Love the description!

Reply:
Fixed!